.github/workflows/packaging.yml

```diff
@@ -65,8 +65,8 @@ jobs:
           file_glob: true
           file: build/linux/*
 
-  macos-net:
-    name: macOS .NET
+  macos:
+    name: macOS
```

Comment:
#20487 split the name `macOS Disk Images` into `macOS .NET` and `macOS Mono` Since we are getting back to a single GH action it might be better to revert to the original name